packages/uniforms-semantic/src/BoolField.tsx

```diff
@@ -15,13 +15,21 @@ const Bool = ({
   required,
   showInlineError,
   value,
+  wrapClassName,
   ...props
 }) => (
   <div
     className={classnames(className, { disabled, error, required }, 'field')}
     {...filterDOMProps(props)}
   >
-    <div className="ui checkbox">
+    <div
+      className={classnames(
+        'ui',
+        wrapClassName,
+        !label && label !== true && 'fitted',
```

Comment:
Is there a way to put `true` in here? I'm pretty sure in this case a default value (e.g. from schema) is taken instead.